com.gooseberrydevelopment.pinepods.yml

```diff
@@ -0,0 +1,44 @@
+id: com.gooseberrydevelopment.pinepods
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: app
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - ../../shared-modules/libappindicator/libappindicator-gtk3-12.10.json
```

Comment:
it won't find this.

Reply:
Fixed the shared-modules reference to use the correct path relative to the Flathub repository root: